src/element.ts

```diff
@@ -88,12 +88,12 @@ export class Element {
 
   protected rendered: boolean;
   protected style?: ElementStyle;
-  protected boundingBox?: BoundingBox;
   protected registry?: Registry;
 
-  protected textFont: Required<FontInfo>;
-  protected text = '';
-  protected textMetrics: TextMetrics = {
+  #textFont: Required<FontInfo>;
```

Comment:
I thought you renamed textFont to fontInfo in other classes. Why not this one?

Reply:
Well, I left it as it is. Element had the internal protected variable textFont and two accessors defined one fontInfo returning FontInfo and one font returning string. I changed textFont to private and used the already existent accessor fontInfo.

Reply:
Might as well have the private instance var match the accessor. Since you no longer want to differentiate between text fonts vs music fonts.... there is no need for it to be called "textFont".